tests/unit/test_adzerk_api.py

```diff
@@ -64,6 +64,14 @@ def test_new_placements(self):
             self.assertEqual(10250, p['networkId'])
             self.assertEqual(1070098, p['siteId'])
             self.assertEqual(20, p['count'])
+            self.assertEqual([5000], p['zoneIds'])
+
+    def test_default_zone(self):
+        api = Api(pocket_id="{123}", placements=mock_spocs_placement)
+        body = api.get_decision_body()
+        self.assertTrue(1, len(body['placements']))
+        for p in body['placements']:
+            self.assertEqual([217995], p['zoneIds'])
```

Comment:
[minor] 
Given that we now have a bunch of constants in tests which are referencing defaults in the production code, should we start a class/module that houses all these constants? We could then do `self.assertEqual([config_defaults.PROD_DEFAULT_ZONE]...)` or something.

Reply:
I was thinking the exact same thing.. Not great to have these "magic numbers" in code. Not in scope for now because I'd like to get this out today.

Reply:
Definitely understand. Approving now.